src/import-data/ImportDataDestination.ts

```diff
@@ -247,7 +252,7 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
   const renderSelectTemplateWorkspace = () =>
     h(Fragment, [
       h2({ style: styles.title }, ['Select a template']),
-      importMayTakeTime && div({ style: { marginBottom: '1rem', lineHeight: '1.5' } }, [importMayTakeTimeMessage]),
+      importMayTakeTime && div({ style: { marginBottom: '1rem', lineHeight: '1.5' } }, [p([importMayTakeTimeMessage])]),
```

Comment:
I don't know how to get to this part of the UI, but I didn't actually change anything.